scripts/base/frameworks/logging/writers/ascii.zeek

```diff
@@ -38,7 +38,7 @@ export {
 	## written into logs.
 	##
 	## This option is also available as a per-filter ``$config`` option.
-	const enable_utf_8 = F &redef;
+	const enable_utf_8 = T &redef;
```

Comment:
Nit: we write UTF8 elsewhere, so would suggest `enable_utf8`.

Reply:
That's an existing option so I'm somewhat leery to change it.